src/main/java/io/jenkins/plugins/orka/OrkaAgent.java

```diff
@@ -46,7 +52,39 @@ public class OrkaAgent extends AbstractCloudSlave {
     private String namePrefix;
     private String jvmOptions;
 
+    private final List<PortMapping> portMappings;
+
     @DataBoundConstructor
+    public OrkaAgent(String name, String orkaCredentialsId, String orkaEndpoint, String vmCredentialsId,
+            String node, String namespace, String namePrefix, String redirectHost, String image,
+            Integer cpu, boolean useNetBoost, boolean useLegacyIO, boolean useGpuPassthrough, 
+            int numExecutors, String host, int port, String remoteFS, boolean useJenkinsProxySettings, 
+            boolean ignoreSSLErrors, String jvmOptions, String memory, String tag, Boolean tagRequired, 
+            List<PortMapping> portMappings)
+            throws Descriptor.FormException, IOException {
+        super(name, remoteFS, new OrkaComputerLauncher(host, port, redirectHost, jvmOptions));
+
+        this.orkaCredentialsId = orkaCredentialsId;
+        this.orkaEndpoint = orkaEndpoint;
+        this.vmCredentialsId = vmCredentialsId;
+        this.namespace = namespace;
+        this.namePrefix = namePrefix;
+        this.node = node;
+        this.image = image;
+        this.cpu = cpu;
+        this.useNetBoost = useNetBoost;
+        this.useLegacyIO = useLegacyIO;
+        this.useGpuPassthrough = useGpuPassthrough;
+        this.useJenkinsProxySettings = useJenkinsProxySettings;
+        this.ignoreSSLErrors = ignoreSSLErrors;
+        this.jvmOptions = jvmOptions;
+        this.memory = memory;
+        this.tag = tag;
+        this.tagRequired = tagRequired;
+        this.setNumExecutors(numExecutors);
+        this.portMappings = portMappings != null ? portMappings : new ArrayList<>();        
+    }
+
     public OrkaAgent(String name, String orkaCredentialsId, String orkaEndpoint, String vmCredentialsId,
```

Comment:
It is a good practice that less specific methods or ctors call the more specific ones instead of repeating the code.
Can we do this here?